docs/src/main/tut/verbs.md

```diff
@@ -0,0 +1,47 @@
+---
+menu: verbs
+weight: 115
+title: REST Verbs
+---
+
+For a REST API, your service will want to support different verbs. Http4s has a
+list of all the [verbs] you're familiar with, and a few more.
+
+```tut:book
+import fs2.Task
+import io.circe.generic._
+import io.circe.syntax._
+import org.http4s._, org.http4s.dsl._
+import org.http4s.circe._
+
+@JsonCodec case class TweetWithId(id: Int, message: String)
+@JsonCodec case class Tweet(message: String)
+
+def getTweet(tweetId: Int): Task[Option[TweetWithId]] = ???
+def addTweet(tweet: Tweet): Task[TweetWithId] = ???
+def updateTweet(id: Int, tweet: Tweet): Task[Option[TweetWithId]] = ???
+
+implicit val tweetWithIdEncoder = jsonEncoderOf[TweetWithId]
+implicit val tweetDecoder = jsonOf[Tweet]
+
+val tweetService = HttpService {
+  case GET -> Root / "tweets" / IntVar(tweetId) =>
+    getTweet(tweetId)
+      .flatMap(_.fold(NotFound())(Ok(_)))
+  case req @ POST -> Root / "tweets" =>
+    req.as[Tweet].flatMap(addTweet).flatMap(Ok(_))
```

Comment:
I think it would be worth referring to `EntityDecoder` here somehow, to make clear that this works for a variety of things.